packages/integration-tests/src/relations/OneToOneReference.test.ts

```diff
@@ -100,22 +100,22 @@ describe("OneToOneReference", () => {
     expect(a1.image.get?.fileName).toEqual("f1");
   });
 
-  it("id fails if not loaded", async () => {
-    await insertAuthor({ first_name: "a1" });
-    const em = newEntityManager();
-    const a1 = await em.load(Author, "1");
-    expect(() => a1.image.id).toThrow("Author:1.image was not loaded");
-    await a1.image.load();
-    expect(a1.image.id).toBeUndefined();
-  });
+  // it("id fails if not loaded", async () => {
+  //   await insertAuthor({ first_name: "a1" });
+  //   const em = newEntityManager();
+  //   const a1 = await em.load(Author, "1");
+  //   expect(() => a1.image.id).toThrow("Author:1.image was not loaded");
+  //   await a1.image.load();
+  //   expect(a1.image.id).toBeUndefined();
+  // });
```

Comment:
delete this?

Reply:
My intention was to restore this pre-merging; went ahead and did that in #213 . It required added a dedicated `LoadedOneToOneReference` that adds the `id` / `isSet` / etc. methods that are now only available on the unloaded `ManyToOneReference`.